README.md

```diff
@@ -135,4 +135,4 @@ Mailing List
 ------------
 
 Our mailing list is open to all:
-[http://groups.google.com/group/cpan-api](http://groups.google.com/group/cpan-api) 
+[http://groups.google.com/group/cpan-api](http://groups.google.com/group/cpan-api)
```

Comment:
I wonder if we should just delete this.  I can't remember the last time someone used the list.  I had actually forgotten that it exists.  Does it still exist?